templates/news_section.html

```diff
@@ -11,35 +11,27 @@
 	<div class="news-list-parent">
 		<div class="news-list">
 			<div class="news-list-content">
-				<div class="news-list-grid">
-					{# Get all news pages #}
-					{% set all_news = section.pages %}
-					{# Separate Japanese and English articles #}
-					{% set ja_news = all_news | sort(attribute="path") | reverse %}
+				{% set all_news = section.pages %}
+				{% set ja_news = all_news | sort(attribute="path") | reverse %}
```

Comment:
if you set `sort_by` in the toml front matter of `news/_index.md`, you don't need to sort it here.

> ```
> # Used to sort pages by "date", "update_date", "title", "title_bytes", "weight", "slug" or "none". See below for more information.
> sort_by = "none"
> ```

See: https://www.getzola.org/documentation/content/section/#front-matter